cluster-provision/fedora/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM kubevirtci/base
```

Comment:
We need a hash here.

Reply:
There is no hash because I built a new base image and uploaded it to my own registry because I dont have permissions to kubevirtci registry

Reply:
I think Quique already wrote it somewhere else, there should be a prow job to push the images (or you need to create a new one). Prow has the credentials, we don't push the images from local.

Reply:
Will do

Reply:
Have run push-base job so we will have a hash soon.

Reply:
sha256:cb9a1cb7740f9b717d4988ffe3d5a7e41f38ed9b2fe9013458b225de19b7c3b4

Reply:
done